templates/common/titleBannerBigRed.html.twig

```diff
@@ -0,0 +1,11 @@
+{% set _hasLink = link is defined %}
+
+<section class="titleBannerNoBackground {{ className | default('') }}">
```

Comment:
I don't think it's useful to create a full new common template for this. 
This kind of component should stay the most generic possible.
In my opinion, it's best to stick with `titleBanner.html.twig` and update the style ( className can be modified later so we can change the background color here).